api/controllers/AddConditionController.js

```diff
@@ -7,7 +7,13 @@
 
 module.exports = {
   create: function (req, res) {
-    res.view('pages/conditions/add');
+    if (!_.has(req.session, 'medicalReport')) {
```

Comment:
These next few lines show up in a whole bunch of places. Is there any way we might either make this a helper, policy, or some kind of middleware to avoid repetition and enable future changes in one place?

Reply:
hmm... true. I had been thinking this stuff is temporary because of the session dependency, so figured whatever, it'll be gone soon .... but now that I think of it, we'll need similar later, just different backend (db vs session) .... and maybe this would make sense in a policy. Let me try something along those lines.

Reply:
Just pushed a change with that bit of code moved out to a policy. Pretty clean I think. Note that I did a rebase and a force-push. You may need to delete your local branch and re-pull? Not sure how that works...